src/Symfony/Component/Form/Tests/Extension/DataCollector/FormDataExtractorTest.php

```diff
@@ -314,12 +314,12 @@ public function testExtractSubmittedDataStoresErrorCause()
           -message: "Foo"
           -messageTemplate: "Foo"
           -parameters: []
-          -plural: null
           -root: "Root"
           -propertyPath: "property.path"
           -invalidValue: "Invalid!"
-          -constraint: null
+          -plural: null
           -code: null
+          -constraint: null
```

Comment:
This change of order is annoying, not sure how we could tackle it.
We could bump the deps, but that's putting the hurdle on end users.
Another idea would be to make the comparison order-independent on 6.4, then the issue will disappear. That's my preference but I didn't check how to do it :)

Reply:
(or revert the changes?)

Reply:
I reverted `src/Symfony/Component/Form/Tests/Extension/DataCollector/FormDataExtractorTest.php` and `src/Symfony/Component/Validator/ConstraintViolation.php`, all green now